data/mods/Megafauna/monstergroups/wilderness.json

```
@@ -2117,22 +2117,23 @@
         "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
       },
       {
+        "group": "GROUP_FROGS_ALL",
         "monster": "mon_frog",
```

```suggestion
        "group": "GROUP_FROGS_ALL",
```
I'm assuming you missed that one?